docs/installation/heroku.md

```diff
@@ -25,10 +25,13 @@ One-click Heroku deployment is available:
 If they are missing, some problem has occured in the previous steps.
 * Add python build pack to the app
     * ```heroku buildpacks:set heroku/python```
-* Add Node.js build pack to the app
-    * ```heroku buildpacks:add --index 2 heroku/nodejs```
```

Comment:
@niranjan94 I was able to deploy it without using it. I think it's from the version 1 when we had both server and frontend. 
Please correct if wrong.

Reply:
Yes nodeJS build pack is not required since we are no longer using node.

Reply:
And it fails as no `package.json` can be found

Reply:
Yes, exactly.